torchvision/transforms/functional_tensor.py

```diff
@@ -1180,3 +1181,146 @@ def gaussian_blur(img: Tensor, kernel_size: List[int], sigma: List[float]) -> Te
 
     img = _cast_squeeze_out(img, need_cast, need_squeeze, out_dtype)
     return img
+
+
+def invert(img: Tensor) -> Tensor:
+    if not _is_tensor_a_torch_image(img):
```

Comment:
Out of topic, maybe we can introduce a decorator with this check to simplify the code ?
```python
@check_type
def invert(img: Tensor) -> Tensor:
    # no more if not _is_tensor_a_torch_image(img): etc
    ...
```

Reply:
Good idea. As this PR is getting already too long, I raised a separate ticket for this. See #3159.